fb-setup-discovery.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+# vim: ft=sh:sw=2:et
+
+set -o pipefail
+
+load os_helper
+load foreman_helper
+
+@test "install discovery plugin" {
+  yum -y install tfm-rubygem-foreman_discovery tfm-rubygem-hammer_cli_foreman_discovery
+}
+
+@test "create discover-host script" {
```

Comment:
You still create /usr/local/bin/discover-host. I don't really see why that's needed.